test/content/ContentFactoryTests.js

```diff
@@ -87,5 +87,49 @@ describe('Content Clone Factory Tests', () => {
             // URIs should not be the same
             expect(await dev1Content.contractUri()).not.equal(await dev2Content.contractUri());
         });
+
+        it('Invalid Create Contract parameters', async () => {
+            // Check initializer parameters
+            var uri1 = "arweave.net/tx-contract-uri";
+
+            await expect(contentFactory.connect(developer1Address).createContracts(developer1Address.address, 1500000, uri1))
+                .to.be.reverted;
+            await expect(contentFactory.connect(developer1Address).createContracts(null, 10000, uri1))
+                .to.be.reverted;
+            await expect(contentFactory.connect(developer1Address).createContracts(ethers.constants.AddressZero, 20000, uri1))
+                .to.be.reverted;
+        });
+
+        it('Update Contracts', async () => {
```

Comment:
Actually, you also need to write tests for "invalid contractFactory.updateContracts() parameters" but it currently doesn't fail. Can you add a null address and interface check for each input in the updateContracts()? 

Then, can you add the invalid parameters as well.